addons/mail/__manifest__.py

```diff
@@ -268,7 +268,6 @@
 
             'mail/static/src/new/**/*',
             ('remove', 'mail/static/src/new/views/**/*'),
```

Comment:
shouldn't this line be renamed to exclude web instead now?

Reply:
Or even better possibly, having the web folder only included into the assets backend bundle instead of the mail bundle?

Reply:
change done